app/src/main/java/org/mozilla/focus/utils/IntentUtils.java

```diff
@@ -33,6 +33,7 @@
     private static final String MARKET_INTENT_URI_PACKAGE_PREFIX = "market://details?id=";
     private static final String EXTRA_BROWSER_FALLBACK_URL = "browser_fallback_url";
     public static final String EXTRA_IS_INTERNAL_REQUEST = "is_internal_request";
```

Comment:
We always add `EXTRA_IS_INTERNAL_REQUEST` flag whenever we add `EXTRA_OPEN_NEW_TAB` flag. Can we get rid of `EXTRA_IS_INTERNAL_REQUEST` and make some change in 
```
private boolean isStartedFromExternalApp() {
```
with a new function 
```
private boolean isFromInternal() {
    return intent != null && intent.getBooleanExtra(IntentUtils.EXTRA_IS_INTERNAL_REQUEST, false); 
}
```
and write some comment there to illustrate this?